src/ftp.rs

```diff
@@ -20,27 +24,125 @@ lazy_static! {
     static ref SIZE_RE: Regex = Regex::new(r"\s+(\d+)\s*$").unwrap();
 }
 
+#[cfg(feature = "secure")]
+lazy_static! {
+    // Shared SSL context
+    static ref SSL_CONTEXT: SslContext = match SslContext::new(SslMethod::Sslv23) {
```

Comment:
From the docs `Sslv23 Support the SSLv2, SSLv3, TLSv1, TLSv1.1, and TLSv1.2 protocols depending on what the linked OpenSSL library supports.`  So I believe this will be ok.